src/main/java/qna/domain/AnswerRepository.java

```diff
@@ -1,12 +1,13 @@
 package qna.domain;
 
-import org.springframework.data.jpa.repository.JpaRepository;
-
 import java.util.List;
 import java.util.Optional;
 
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.query.Param;
+
 public interface AnswerRepository extends JpaRepository<Answer, Long> {
-    List<Answer> findByQuestionIdAndDeletedFalse(Long questionId);
+	List<Answer> findByQuestion_IdAndDeletedFalse(@Param(value = "questionId") Long questionId);
```

Comment:
```suggestion
	List<Answer> findByQuestionIdAndDeletedFalse(@Param(value = "questionId") Long questionId);
```
뭔가 `_`가 오타로 들어간 것 같습니다 🙏